includes/class-wc-structured-data.php

```diff
@@ -29,7 +29,6 @@ public function __construct() {
 		add_action( 'woocommerce_before_main_content', array( $this, 'generate_website_data' ), 30 );
 		add_action( 'woocommerce_breadcrumb', array( $this, 'generate_breadcrumblist_data' ), 10 );
 		add_action( 'woocommerce_single_product_summary', array( $this, 'generate_product_data' ), 60 );
-		add_action( 'woocommerce_review_meta', array( $this, 'generate_review_data' ), 20 );
```

Comment:
We could wait a little, we had several changes in schema since is created, I can see some change of getting back to that old format, even have both, since display schema individually for all reviews. So it's possible that we may get back to that code. So at least wait a little before try deprecated it.